content/en/docs/examples/bookinfo/index.md

```diff
@@ -192,22 +192,16 @@ versions, called *subsets*, in [destination rules](/docs/concepts/traffic-manage
 
 Run the following command to create default destination rules for the Bookinfo services:
 
-* If you did **not** enable mutual TLS, execute this command:
-
-    {{< tip >}}
-    Choose this option if you are new to Istio and are using the `demo`
-    [configuration profile](/docs/setup/additional-setup/config-profiles/).
-    {{< /tip >}}
-
-    {{< text bash >}}
-    $ kubectl apply -f @samples/bookinfo/networking/destination-rule-all.yaml@
-    {{< /text >}}
-
-* If you **did** enable mutual TLS, execute this command:
+{{< text bash >}}
+$ kubectl apply -f @samples/bookinfo/networking/destination-rule-all.yaml@
+{{< /text >}}
 
-    {{< text bash >}}
-    $ kubectl apply -f @samples/bookinfo/networking/destination-rule-all-mtls.yaml@
-    {{< /text >}}
+{{< tip >}}
+You can enforce mutual TLS using the `samples/bookinfo/networking/destination-rule-all-mtls.yaml`
+destination rule.  However, the `default` and `demo` [configuration profiles](/docs/setup/additional-setup/config-profiles/) have [auto mutual TLS](/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls) enabled by default.  If a TLS context is not provided in a
+destination rule, Istio configures client proxies to send mutual TLS traffic to injected proxies
+automatically.
+{{< /tip >}}
 
 Wait a few seconds for the destination rules to propagate.
```

Comment:
Do you really need to wait a few seconds?

Reply:
Not really, since it only matters when route rules are applied, which isn't being done immediately afterwords. That said, the general rule is to wait for config to propagate before proceeding to whatever is done next.

Reply:
Most cases, in the time it takes for a user to copy and past the next set of instructions, things will have propagated by then.  However, I think it is good practice to leave comments like this in place to reiterate that it may take a few seconds for istio routing/authentication rules to go into effect.